license_test_config.yml

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would change the file name. This suggest a test file. For example: check_license_config.yml

Reply:
ok